src/components/container/ContainerComponent.tsx

```diff
@@ -144,14 +156,22 @@ export function ContainerComponent({
   );
 }
 
-function getCursor(mode: RoiMode, altKey: boolean, action: RoiAction) {
+function getCursor(
+  mode: RoiMode,
+  altKey: boolean,
+  action: RoiAction,
+  lockPan: boolean,
+) {
+  if (lockPan) return 'default';
```

Comment:
It's more complicated than that...

For example now when the mode is 'draw', the default cursor can appear which is wrong